library/ecp_invasive.h

```diff
@@ -40,8 +40,28 @@ typedef enum {
     MBEDTLS_ECP_MOD_SCALAR
 } mbedtls_ecp_modulus_type;
 
+/* Provide a commented-out definition so that `check_names.py` knows that
+ * it's not a typo.
+ * MBEDTLS_ECP_WITH_MPI_UINT must not be defined within config files, but
+ * only on the command line.
+ */
+//#define MBEDTLS_ECP_WITH_MPI_UINT
+
+typedef enum {
+    MBEDTLS_ECP_VARIANT_NONE            = 0,
+    MBEDTLS_ECP_VARIANT_WITH_MPI_STRUCT = 1,
+    MBEDTLS_ECP_VARIANT_WITH_MPI_UINT   = 2
```

Comment:
It's reasonable to have `= 0` for the first entry in an enum (even though it's the default) but if subsequent numbers are just incrementing by `1` then don't have `= n` for those, as that's the default, and in larger enums can lead to duplication etc